Common/Product/ReplWindow/Repl/OutputBuffer.cs

```diff
@@ -298,7 +294,7 @@ struct OutputEntryProperties {
             }
         }
 
-        enum OutputEntryKind {
+        public enum OutputEntryKind {
```

Comment:
internal

also check usages and see if it makes sense to move into ReplWindow altogether